buildout.cfg

```diff
@@ -14,17 +14,19 @@ parts +=
     node
     omelette
     rebuild_i18n-sh
+    robot
+    webpack
+    webpack_env
+    npm_watch
+    npm_dev
+    npm_build
 
 [code-analysis]
 recipe = plone.recipe.codeanalysis[recommended]
 directory = ${buildout:directory}/src/collective/upload
 clean-lines = True
-clean-lines-exclude = ${buildout:directory}/src/collective/upload/static/vendor
-csslint = True
```

Comment:
AFAIK code-analysis works just into the directory mentioned, and webpack folder is outside this.

And there is no point to check the JS and CSS generated by webpack.